### DISCUSSION TITLE
Why does Obsrv depend on Lern

### DISCUSSION THREAD

**Discussion by pramodkvarma** [Sunbird-Obsrv/Community#7] [Issues] [answered]
@rhwarrier @SanthoshVasabhaktula @maheshkumargangula
I am surprised that SB Obsrv has a hard dependency on SB Lern. That should be fixed given denorm from Lern should be an option and others adopting may use their own metadata/masterdata services for denorm. 

Can you clarify? Thanks.

**Comment by anandp504** [comment, marked as answer]
@pramodkvarma Sunbird Obsrv depends on the user-read-api `/learner/private/user/v1/read` for the denormalization. I will be updating the documentation with the response structure required for the denormalization of the user metadata so that we can remove the hard dependency on Sunbird Lern.

**Reply by pramodkvarma** [reply]
Great! Create the API interfaces and make these pluggable. Thanks @anandp504.
